packages/gatsby/src/utils/eslint-rules/__tests__/limited-exports-page-templates.ts

```diff
@@ -78,6 +78,12 @@ describe(`no-anonymous-exports-page-templates`, () => {
       test({
         code: `import { graphql, Link } from "gatsby"\nconst Template = () => {}\nexport const query = graphql\`test\`\nexport default Template\nexports.config = () => { return ({ params }) => { defer: true }}`,
       }),
+      test({
```

Comment:
This test is missing cases of defining the `Head` directly in the page (it only re-exports it from somewhere). As function/arrow function etc.

See the other exports for inspiration

Reply:
Good catch!